src/components/Header.module.css

```
@@ -75,6 +75,28 @@
+  transform: translateX(-50%);
+  background: var(--ink);
+  color: #fff;
+  font-size: 13px;
+  padding: 9px 18px;
+  border-radius: var(--radius-pill);
+  white-space: nowrap;
+  pointer-events: none;
+  animation: fadeInOut 2.5s ease forwards;
+}
+
+@keyframes fadeInOut {
```

Stylelint (17.14.0)

[error] Expected keyframe name "fadeInOut" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)